pkg/cmd/cli/describe/describer.go

```diff
@@ -283,17 +283,16 @@ func (d *BuildConfigDescriber) Describe(namespace, name string) (string, error)
 		if len(builds.Items) == 0 {
 			return nil
 		}
-		fmt.Fprintf(out, "Builds(Name,Status,Creation Time)\n")
+		fmt.Fprintf(out, "Builds(Name,Status,Duration,Creation Time)\n")
 		sortedBuilds := sortableBuilds(builds.Items)
 		sort.Sort(sortedBuilds)
-		c := 0
 		for i := range sortedBuilds {
 			// iterate backwards so we're printing the newest items first
 			build := sortedBuilds[len(sortedBuilds)-1-i]
-			fmt.Fprintf(out, fmt.Sprintf("- %s %s %v\n", build.Name, build.Status, build.CreationTimestamp.Rfc3339Copy().Time))
-			c++
+			duration := build.CompletionTimestamp.Rfc3339Copy().Time.Sub(build.StartTimestamp.Rfc3339Copy().Time)
```

Comment:
i'd prefer #1.  Replacing completionTimestamp doesn't make sense as we are calculating various "durations" depending on the state of the build (duration waiting for a pod, duration actually running) and we want a duration value for running builds, not just completed ones.

Also be aware that the web console uses all these timestamp values and does similar logic, so if you change the behavior, the webconsole should be updated as well.